tests/bin/sessions.test.ts

```diff
@@ -163,7 +163,11 @@ describe('Session Token Refreshing', () => {
       global.failedConnectionTimeout,
     );
 
-    test('Parallel processes should not refresh the session token', async () => {
+    // Currently requests to refresh the session token will not be dropped if
```

Comment:
@joshuakarp @emmacasolin please look into creating a new issue for this.

Reply:
Issue created for this here: https://github.com/MatrixAI/js-polykey/issues/303